tf_encrypted/protocol/pond.py

```diff
@@ -989,6 +989,142 @@ def dispatch(self, base_name, *args, container=None, **kwargs):
         else:
             raise TypeError("Don't know how to {}: {}".format(base_name, [type(arg) for arg in args]))
 
+    @memoize
+    def zeros_private(
```

Comment:
thinking maybe we want to combine these into one function?  I know the normal method of dispatching won't work the same way here, but the code is so similar that it feels like it'd be better to pass a kwarg in that determines if the output is public, private, or masked and have it return the corresponding tensor type